main.cpp

```diff
@@ -73,12 +24,31 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+extern void PrintReferenceManual();
+
 int main() {
-    testNumberToPair(4, TelCoColorCoder::WHITE, TelCoColorCoder::BROWN);
-    testNumberToPair(5, TelCoColorCoder::WHITE, TelCoColorCoder::SLATE);
+    TelCoColorCoder::InitializeDefaultEnglishLabels();
+
+    // Test valid and invalid pair numbers
+    try {
+        testNumberToPair(4, TelCoColorCoder::WHITE, TelCoColorCoder::BROWN);
+        testNumberToPair(5, TelCoColorCoder::WHITE, TelCoColorCoder::SLATE);
+        testNumberToPair(26, TelCoColorCoder::RED, TelCoColorCoder::BLUE); // Invalid pair number
+    } catch (const std::out_of_range& e) {
```

Comment:
Normally we don't use catch around tests, because it can mask the error, causing the github action / jenkins to report a false-positive. Better to let a failure abort the run, so we can check the final result without having to look through the console output.